From: john cooper <john.cooper@redhat.com>
Date: Sat, 4 Apr 2009 16:29:28 -0400
Subject: [mm] mmu_notifier: set CONFIG_MMU_NOTIFIER to y
Message-id: 49D7C328.6060506@redhat.com
O-Subject: [RHEL5.4 PATCH] implement mmu_notifier mechanism V3, [4/5] BZ#485718
Bugzilla: 485718
RH-Acked-by: Rik van Riel <riel@redhat.com>
RH-Acked-by: Rik van Riel <riel@redhat.com>

BZ#485718  https://bugzilla.redhat.com/show_bug.cgi?id=485718

--
john.cooper@redhat.com

Subject: set CONFIG_MMU_NOTIFIER default to y

This will be needed because KVM won't be packaged on the kernel package.

diff --git a/mm/Kconfig b/mm/Kconfig
index 2047f46..719ef3c 100644
--- a/mm/Kconfig
+++ b/mm/Kconfig
@@ -161,3 +161,4 @@ config RESOURCES_64BIT
 
 config MMU_NOTIFIER
 	bool
+	default y
